cdp_client/cdp.py

```diff
@@ -243,7 +246,8 @@ def fetch_structure():
 
         def fetch_value(node):
             if self._value_subscriptions:
-                self._connection.send_value_request(self._id())
+                max_fs = max(self._value_subscriptions, key=lambda e: e[1])
```

Comment:
Should have a test that on multiple request with different fs max fs will be used for all subscriptions

Reply:
added